src/javase02/task02/Worker.java

```diff
@@ -0,0 +1,57 @@
+package javase02.task02;
+
+import java.util.Scanner;
+
+public class Worker {
+    String name;
+    int numberOfChancellery;
+    Chancellery[] chancelleries;
+    //Chancellery[] chancelleries = new Chancellery[numberOfChancellery];
+    static Scanner input = new Scanner(System.in);
+    private int calculatePriceOfChancellery(Worker obj){
+        int summ=0;
+        for (int i =0;i<numberOfChancellery;i++) summ+=obj.chancelleries[i].price;
+        return summ;
+    }
+
+    Worker(String name1, int number){
```

Comment:
name1 is bad name

Reply:
I know. I can use _this_-keyword in this constructor. Is it a good idea?

Reply:
yes